### ISSUE TITLE
Incorrect name & role is defined for ‘Formatted data’ & ‘Download for excel’ toggle buttons.

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#87801]
### What happened?

Aria label is defined as ‘Toggle switch’ & role is defined as ‘Checkbox’ for the ‘Formatted data’ & ‘Download for excel’ toggle buttons.  

https://github.com/grafana/grafana/assets/93735775/4c0a9766-6d8a-42c0-90a9-d2f1894870ca

### What did you expect to happen?

Respective name of the toggle switch should be defined as aria label for the control & role should be defined as ‘Toggle button’ for ‘Formatted data’ & ‘Download for excel’ toggle buttons. 

### Did this work before?

No

### How do we reproduce it?

1.Open the URL [Grafana Play Home - Dashboards - Grafana](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1) in edge dev browser. 

2.’Grafana Play Home-Dashboards Grafana’ page will open. 

3.Navigate to ‘…’ (More options) button present for any of the graph/table & activate it. 

4.Navigate to ‘Inspect’ option & activate it. 

5.’Inspect’ dialog will open. 

6.Navigate to ‘Data’ tab & select it. 

7.Navigate to ‘Data options’ button & expand it. 

8.Turn on Narrator using ‘Ctrl+Windows+Enter’ key. 

9.Navigate to ‘Formatted data’ & ‘Download for excel’ toggle buttons via tab key & observe the screen reader narration. 

### Environment (with versions)?

Grafana:  v11.1.0-69051 (9d44c8e8cf) 
OS: Version 24H2 (OS Build 26100.1) 
Browser: Version 125.0.2506.0 (Official build) dev (64-bit) 

### Grafana platform?

None
